### DISCUSSION TITLE
syntax highlighting

### DISCUSSION THREAD

**Discussion by ryanshrott** [Zero6992/chatGPT-discord-bot#261] [Ideas]
is it possible to do syntax highlighting in discord for code outputs?

**Comment by Zero6992** [comment]
GPT-4 has better support for syntax highlighting compared to GPT-3.5, GPT-3.5  not always recognize its coding language

**Reply by ryanshrott** [reply]
> GPT-4 has better support for syntax highlighting compared to GPT-3.5, GPT-3.5  not always recognize its coding language

When I use official chat gpt web app, I can see highlighted code always using gpt 3.5. There is also a copy button for the code and the code is highlighted for readability. Any way to do this in discord?

**Reply by Zero6992** [reply]
> When I use official chat gpt web app, I can see highlighted code always using gpt 3.5. There is also a copy button for the code and the code is highlighted for readability. Any way to do this in discord?

GPT-3.5 will provide incorrect coding language about the code block sometimes, and Discord may not support the syntax highlighting for that language,  so the code block would not have highlighting

The copy button cannot be implemented on Discord
